netbox/ipam/tables/ip.py

```diff
@@ -374,7 +374,7 @@ class IPAddressTable(TenancyColumnsMixin, NetBoxTable):
         verbose_name='NAT (Outside)'
     )
     assigned = columns.BooleanColumn(
-        accessor='assigned_object_id',
+        accessor='assigned_object',
         linkify=True,
```

Comment:
Unfortunately this breaks ordering for the column (because the accessor is no longer tied to a discrete field). What we can do instead is pass a callable to `linkify` which will dynamically resolve the URL for the assigned object.

```suggestion
        accessor='assigned_object_id',
        linkify=lambda record: record.assigned_object.get_absolute_url(),
```